packages/blocks/src/database-block/data-view/column/presets/progress/define.ts

```diff
@@ -25,7 +25,13 @@ export const progressColumnModelConfig = progressColumnType.modelConfig<number>(
     cellToJson: data => data ?? null,
   }
 );
+
 progressColumnModelConfig.addConvert('rich-text', (_column, cells) => ({
   column: {},
   cells: cells.map(v => new Text(v?.toString()).yText),
 }));
+
+progressColumnModelConfig.addConvert('number', (_column, cells) => ({
+  column: { decimal: 0 },
```

Comment:
What does prop decimal store🤔

Reply:
It's used to record the number of decimal places, but it's not currently in use.